zhaquirks/centralite/3130.py

```diff
@@ -40,11 +40,8 @@ class CustomPowerConfigurationCluster(PowerConfigurationCluster):
     """Custom PowerConfigurationCluster."""
 
     cluster_id = PowerConfigurationCluster.cluster_id
-    MIN_VOLTS = 21
-    MAX_VOLTS = 31
-
-    def _update_attribute(self, attrid, value):
-        super()._update_attribute(attrid, value)
+    MIN_VOLTS = 2.1
+    MAX_VOLTS = 3.1
```

Comment:
Just override the min

Reply:
So my follow up question to the 2.8v question... should we be removing these values if its capped at 2.8 now?

Reply:
yeah, the values above 3.0 aren’t really realistic. The batteries are rated at 3.0, if anything... set them to 3.0.

Reply:
Right, but my question is whether these devices "report" values > 3.0. I am pretty indifferent as i do not have these devices, but i am not wanting to improperly scale the battery when it does report higher values. I suppose in the grand scheme of things, it will only alter the scale and be at 100% longer.

Reply:
Right, it really shouldn’t matter

Reply:
:+1:

Reply:
Fixed. Set to 3.0